src/views/PeopleSearch/components/PeopleSearchResult/index.js

```diff
@@ -1,227 +1,239 @@
-import React, { Component } from 'react';
+import React, { useEffect, useState } from 'react';
 import IMG from 'react-graceful-image';
 import { Typography, Grid, Divider } from '@material-ui/core';
 import PropTypes from 'prop-types';
-import user from 'services/user';
 import { Link } from 'react-router-dom';
+import userService from 'services/user';
+import handleViewport from 'react-in-viewport';
 
 import './peopleSearchResult.css';
 
-export default class PeopleSearchResult extends Component {
-  constructor(props) {
-    super(props);
+/*Const string was created with https://png-pixel.com/ .
+ *It is a 1 x 1 pixel with the same color as gordonColors.neutral.lightGray (7/9/21)
+ *Although this doesn't use the gordonColors themes directly,
+ *the end result is much cleaner and faster than using the placeholderColor tag of react-graceful-image.
+ */
+const GORDONCOLORS_NEUTRAL_LIGHTGRAY_1X1 =
+  'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mN8/erVfwAJRwPA/3pinwAAAABJRU5ErkJggg==';
+const JPG_BASE64_HEADER = 'data:image/jpg;base64,';
 
-    this.state = {
-      avatar: null,
-      prefImage: null,
-      defImage: null,
-    };
-  }
+const PeopleSearchResultBlock = ({ Person, size, lazyImages, inViewport }) => {
+  const [avatar, setAvatar] = useState(GORDONCOLORS_NEUTRAL_LIGHTGRAY_1X1);
+  const [hasBeenRun, setHasBeenRun] = useState(false);
+  const [fullName, setFullName] = useState('');
+  const [nickname, setNickname] = useState('');
+  const [personClassJobTitle, setPersonClassJobTitle] = useState('');
+  const [personMailLocation, setPersonMailLocation] = useState('');
 
-  componentDidUpdate(newProps) {
-    if (this.props.Person.AD_Username !== newProps.Person.AD_Username) {
-      this.loadAvatar();
+  useEffect(() => {
+    if (lazyImages === false) {
+      loadAvatar();
     }
-  }
```

Comment:
It was never possible for this to happen, I think this was copied boilerplate code? I don't have an equivalent in the functional component.